cookbook-react/src/recipes/spinners/SpinkitSpinner/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import SpinkitSpinner, { SpinnerProps } from 'react-spinkit';
+
+import { COLOR_SPINNER, SPINNER_DEFAULT } from './constants';
+import styles from './styles.module.scss';
+
+interface LoadingProps extends SpinnerProps {
+  containerClassName: string;
+}
+
+// Check http://kyleamathews.github.io/react-spinkit/ for all possible spinner styles
+function Loading({
```

Comment:
Can you name the recipe and folder Loading?
Folder name and component name should match, it's easier to find the definition this way.